src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,19 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
 public class HelloWorld {
+    private static final int NUMBER_OF_FIGURES = 6;
 
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[NUMBER_OF_FIGURES];
+        for (int i = 0; i < figures.length; i++) {
+            if (i >= figures.length / 2) {
+                figures[i] = figureSupplier.getDefaultFigure();
+            } else {
+                figures[i] = figureSupplier.getRandomFigure();
+            }
+            System.out.println(figures[i].draw());
```

Comment:
The `draw()` method seems to be called from within a `System.out.println()` statement. Ensure that the `draw()` method is not returning a `String` but instead is responsible for drawing the figure. The method name suggests it should be an action rather than a value-returning operation.